spec/models/raw_input_changes_spec.rb

```diff
@@ -150,7 +150,7 @@
             organization_name: 'Best Gallery',
             phone_number: '1-800-123-4567',
             tag_names: 'invalid',
-            website: 'examplecom'
+            website: 'example.com'
```

Comment:
I could see this causing more work for an upload user if they had a bad website and a bunch of invalid data. But let's hold off and see if it's a problem until we refactor.